src/search-box/search-box.ts

```diff
@@ -43,7 +43,6 @@ import { noop } from '../utils';
           [class]="cx('submit')"
           type="submit"
           title="{{submitTitle}}"
-          (click)="handleSubmit($event)"
```

Comment:
On a side not (and for another PR), this should also be modified for the `reset` event. Right now, we call `handleReset()` when we click on the reset button, while it should be handled in the `form` element:

```html
(reset)="handleReset($event)"
```

I'm assuming Angular handles that.
Reference: https://angular.io/api/forms/NgForm#onreset

Reply:
Actually you're looking at the documentation of the `NgForm` directive. We use the native form element here.

You're right, we can handle it with events for more consistency, like we do it for submit
In the case of `submit`, using events allows us to make submission by hitting `Enter`.
Do you know of any similar benefit when using the `reset` event ?

Reply:
I am fairly sure there's a way to clear a form with assistive technologies. This would not trigger the button click, but would the form reset

Reply:
I was pointing that out because React handles it, I have no idea about Angular.

Reply:
bubbling events is part of DOM, not react or angular :)

Reply:
ok, i'll change that to an event. thank you both!

Reply:
It's working perfectly in the browser but can't get to work in the test env and i'd rather not add something I can't test.
I filed an Jira ticket so that I can look at this later.